applications/launchpad/gui-react/src/containers/SettingsContainer/styles.ts

```diff
@@ -9,6 +9,7 @@ export const MainContainer = styled.div`
 export const MainContentContainer = styled.div`
   display: flex;
   flex-grow: 1;
+  max-height: calc(100% - 70px);
```

Comment:
where does the 70px come from? please add at least a const with meaningful name

Reply:
Done! 
Clarification: It keeps the height of the bottom bar. If not set or smaller value used, then we have this:

![image](https://user-images.githubusercontent.com/11715931/172577328-0e5533b5-787c-4751-b304-4a9fe578f926.png)

_It happens when the tab content is larger than the modal height_